### ISSUE TITLE
justpy 0.11 up compatibility

### ISSUE THREAD

**Issue by WolfgangFahl** [zauberzeug/nicegui#168]
```bash
nicegui 0.9.25 requires justpy<0.3.0,>=0.2.3, but you have justpy 0.11.2 which is incompatible.
```
justpy is moving on fast in the last few weeks (currently at version 0.11 ) and i'd love to make sure nicegui and justpy stay compatible. Currently nicegui seems to have a redundant copies of the static files of justpy. So i fear quite a few of the refactorings of justpy  won't apply to nicegui yet.
Especially the templates/js directory content has been refactored - there is a justpy_core.js which handles the javascript side.

**How should we proceed to  keep the libraries in sync with each other?**

**Comment by rodja** [comment]
Indeed JustPy has changed quite a bit. We have sticked to v0.2.x for now because of the many unforeseeable changes and unsolved issues like serving behind a reverse-proxy. Adaptation to 0.11 and later may still be a good intermediate step. But as we are currently working on v1.0 it is not on our priority list. Maybe someone from the community would like to take on the challange?

**Comment by elimintz** [comment]
@rodja You would need someone intimately familiar with the Nicegui and Justpy code to take on the challenge. Realistically, I think that means that your group has to take the lead in this. I think it is important to keep the libraries in sync and eventually have Nicegui integrated with the Justpy CI. @WolfgangFahl has a few ideas and can chime in. 

Also, if you would like to be able to provide more input to the Justpy development process, Wolfgang would be happy to interact with you more frequently.

**Comment by rodja** [comment]
With [NiceGUI 1.0](https://github.com/zauberzeug/nicegui/releases/tag/v1.0.0) we had to let go of JustPy. We felt the need to use Quasar 2 and Vue 3. That proved to be difficult with JustPy (see https://github.com/justpy-org/justpy/issues/460). Also we saw other architectural benefits plus the chance to integrate with FastAPI. Our favorite backend framework.

Thank you, JustPy, for providing the core engine for NiceGUI until now. We are super grateful for the bootstrapping. Without JustPy, NiceGUI would never have been possible. Special thanks to @elimintz as the inventor of JustPy. You showed us the way. And thanks @WolfgangFahl for your dedication to the project. We still think JustPy is a great framework and we are eager to see how it further evolves.
